senders/direct.go

```diff
@@ -47,6 +48,13 @@ func NewDirectSender(cfg *DirectConfiguration) (Sender, error) {
 		internal.SetPrefix("~sdk.go.core.sender.direct"),
 		internal.SetTag("pid", strconv.Itoa(os.Getpid())),
```

Comment:
Even more... we can put the version as tag here, like the pid, that way we don't need to report a new metric.

Reply:
We thought about emitting this as a tag and it was decided that a metric would be more useful going forward like it can be used to create a chart on dashboard etc.